content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Implementation status"
+linkTitle: "Implementation status"
+weight: 8
+---
+
+### Physical types
+
++-------------------------------------------+-------+--------+--------+-------+-------+
```

Comment:
Is this standard Markdown markup for tables? Github doesn't seem to recognize it (see what happens here if you ask to "view" the file).

Reply:
I’ll check and convert it to the correct format. It was copy pasted from arrow-site